docs/features/techdocs/creating-and-publishing.md

```diff
@@ -79,6 +79,8 @@ plugins:
   - techdocs-core
 ```
 
+The plugins section above is optional. Backstage automatically adds the `techdocs-core` plugin to the mkdocs file if it is missing. This functionality can be turned off with a configuration option in Backstage.
```

Comment:
Maybe make this a "note" to give it a bit more attention? And also point to the docs of how its turned off. Something like:

```markdown
 > Note - The plugins section above is optional. Backstage automatically adds the `techdocs-core` plugin to the 
 > mkdocs file if it is missing. This functionality can be turned off with a [configuration option](./configuration.md) in Backstage.
```

Reply:
That makes sense yeah. Updated.